Lib/test/support/_interpreters.py

```diff
@@ -0,0 +1,235 @@
+"""Subinterpreters High Level Module."""
+
+import logging
+import _xxsubinterpreters as _interpreters
+
+__all__ = ['Interpreter', 'SendChannel', 'RecvChannel',
+            'is_shareable', 'create_channel',
+            'list_all_channels', 'get_current',
+            'get_main', 'create']
+
+
+def create(*, isolated=True):
+    """ create() -> Interpreter
+
```

Comment:
For the docstrings in the Python module you don't need to document the signature.